pages/login/github.tsx

```diff
@@ -0,0 +1,33 @@
+import { GetServerSideProps } from 'next';
+import Main from '../../src/components/layout/Main';
+import { LOGIN_PROVIDERS_URL } from '../../src/env';
+
+export default function AuthReturnPage({data}) {
+  // TODO show either success or failure message
+  // TODO store logged-in user info client-side
+  return (
+    <Main>
+      {data}
+    </Main>
+  )
+}
+
+export const getServerSideProps: GetServerSideProps = async (context) => {
+  // Frontend sends returned parameters to backend to complete the flow
+  const res = await fetch(`${LOGIN_PROVIDERS_URL}/github`, {
```

Comment:
We realised because this is being done SSR here (due to CORS issues), the session cookies aren't working as expected and status 500 was always being returned. Once https://github.com/flathub/backend/issues/108 is resolved I will then push the POST request to client-side so that this (hopefully) starts working.

Depending on how fast I'm moving I may also redo the redirects themselves to be client-side, but that's not super important.